subcommands/waves/init.go

```diff
@@ -0,0 +1,152 @@
+package waves
+
+import (
+	"fmt"
+	"strconv"
+	"time"
+
+	"github.com/docker/go/canonical/json"
+	"github.com/sirupsen/logrus"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+
+	"github.com/foundriesio/fioctl/client"
+	"github.com/foundriesio/fioctl/subcommands"
+	tuf "github.com/theupdateframework/notary/tuf/data"
+)
+
+func init() {
+	initCmd := &cobra.Command{
+		Use:   "init <wave> <version> [<tag>]",
+		Short: "Create a new wave for a given targets version.",
```

Comment:
"for a given Target's version"

Reply:
ack

Reply:
After some thought either way seems confusing.
I rephrased this to be more clear: `Create a new wave from targets of a given version.`

Reply:
https://github.com/foundriesio/fioctl/pull/82/commits/e03c23a3501c7d4b5462266d00b37f28460de38a

Reply:
that's much better.